kuma/users/tests/test_views.py

```diff
@@ -1271,6 +1276,14 @@ def test_matching_accounts(self):
                                                       user=octocat3)
         response = self.client.get(self.signup_url)
         assert list(response.context['matching_accounts']) == [social_account]
+        parsed = pq(response.content)
+        # li with class=exists is rendered with a strikeout, to suggest to the
+        # user that signup may fail and they should use account recovery.
+        li_exists = parsed.find('ul.choices li.exists')
```

Comment:
nit: If you use `li_exists = parsed.find('ul.choices li.exists').items()` you get a generator which you can use like jQuery. E.g.

```python
li_exists = parsed.find('ul.choices li.exists').items()
assert len(li_exists) == 1
email_input = li_exists('input[type=radio]')
...
```